applications/zeeve_larch_phase_2.md

```diff
@@ -0,0 +1,209 @@
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/ghan-vashishtha/
+- https://www.linkedin.com/in/sankalp-sharma-502008139/
+- https://www.linkedin.com/in/jastisriradheshyam
+- https://www.linkedin.com/in/supravat03
+
+## Development Status :open_book:
+
+- **Project Link**: [Zeeve-App/larch: Zeeve Larch is a GUI frontend for Zombienet CLI](https://github.com/Zeeve-App/larch)
+- **Blog**: [Zeeve's Larch: Polkadot/Substrate Zombienet Automation and Monitoring with an Intuitive GUI Tool](https://www.zeeve.io/blog/zeeves-larch-polkadot-substrate-zombienet-automation-and-monitoring-with-an-intuitive-gui-tool/)
+- **Previous Grant**: [Grants-Program/applications/Zeeve_Parachain_deployment_zoombienet_testing_automation.md at master · w3f/Grants-Program](https://github.com/w3f/Grants-Program/blob/master/applications/Zeeve_Parachain_deployment_zoombienet_testing_automation.md)
+- Automated Polkadot deployments including validator nodes, archive nodes [Polkadot Node | Launch Your Polkadot Node With Zeeve in Minutes](https://www.zeeve.io/blockchain-protocols/deploy-polkadot-node/)
+- Automated Kusama deployments including validator nodes, archive nodes [Kusama | Blockchain Deployment and Management Platform | Zeeve](https://www.zeeve.io/blockchain-protocols/deploy-kusama-node/)
+- Created a relay chain on substrate with some customisations done at the core to accommodate the tokenomics and custom reward mechanism [AXIA Blockchain Portal](https://axscan.axiacoin.network/#/explorer)
+- Published a [blog post](https://www.zeeve.io/blog/how-to-use-substrate-to-create-parachains-in-polkadot/) about the usage and implementation of parachains
+- Spoke with David Hawig, Richard Casey and Gautam Dhameja from the Parity team regarding the development of Zeeve and the Web3 Grant application
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 6 months
+- **Full-Time Equivalent (FTE):**  4
+- **Total Costs:** 24,000 USD
+
+### Milestone 1 — Monitoring Support for K8 deployments & Global config
+
+- **Estimated duration:** 45 Days
+- **FTE:**  4
+- **Costs:** 7,500 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide the **tutorial** to that explains how a user can create a network configuration and start the network |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 1. | Monitoring for k8 based deployment | We will add Monitoring support to kubernetes based deployment |
+| 2. | Zombienet global parameters support | Add All possible Global parameters in template creation corresponding to the supported Zombienet version [settings](https://paritytech.github.io/zombienet/network-definition-spec.html#settings) |
```

Comment:
Could you list all parameters what will be supported in addition to the current larch version?

Reply:
The link contains the list of global parameters (settings section), they may change at time of final commit based on Zombienet version, do you want to specify all of the them here for current version ?